docs/cloud/ops/README.md

```diff
@@ -0,0 +1,51 @@
+# Event Store Cloud Jobs
+
+The Event Store Cloud runs scheduled jobs on the user's behalf. All of these jobs consist of a human readable description and a schedule.
+
+Currently the following jobs are supported: 
+* [Scheduled backups](./scheduled_backups.md)
+
+## Job Schedules
+
+The schedule format used by the Event Store Cloud CLI tool and API is a simplified subset of [cron](https://en.wikipedia.org/wiki/Cron#Overview), in the future we may support more cron features.
+
+The supported subset is:
+
+* for the first field, minute:
+    * a wildcard `*`
+    * a number between `0` and `59` (inclusive). 
+    * a rate, written as a wildcard divided by a number. E.g:  `*/15`
+* fro the second field, hour:
```

Comment:
For

Reply:
Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.